### DISCUSSION TITLE
Custom Class Container

### DISCUSSION THREAD

**Discussion by pandermatt** [vuejs/vitepress#3608] [Ideas]
Now that the home layout supports markdown, the text is a bit too wide compared to other pages.

Maybe it would be useful to use custom containers with custom classes like this (Something similar was already mentioned in #548):
```md
::: {my-class}
This is a custom class div
:::
```

since this will break the MD formatting:
```md
<div class="my-class">
This is a custom class div
</div>
```

```css
.my-class {
	max-width: 800px;
}
```

**Comment by brc-dd** [comment]
> will break the MD formatting

It won't, just separate your html and line with a new line:

```html
<div class="my-class">

*markdown works here*

</div>
```

**Reply by pandermatt** [reply]
Thank you!
